examples/landsat.ipynb

```diff
@@ -225,14 +225,13 @@
     "band_map = hv.HoloMap({i: hv.Image(band) for i, band in enumerate(bands)})\n",
     "\n",
     "def spectrum(x, y):\n",
+    "    point = gv.Points([(x, y)], crs=ccrs.GOOGLE_MERCATOR)\n",
+    "    point = gv.operation.project_points(point, projection=ccrs.PlateCarree())\n",
+    "    label = 'Lon: %.3f, Lat: %.3f' % tuple(point.array()[0])\n",
     "    try: \n",
```

Comment:
I should probably have explained why I reorganized that function the way I did; it was to avoid starting up with bogus labels like:  

![image](https://user-images.githubusercontent.com/1695496/31182382-588b5212-a8e9-11e7-8714-ce7ed80b12de.png)

It's ok to leave the lat,lon coordinates up there when the curve disappears, as you do now, but I don't want it to start with nan,nan shown, because that's how the page will look on a website when it's exported statically.

Reply:
Without an initial label the title initializes inside the plot for me, which I think is worse than showing the nans.

Reply:
Seems like a bug report to file somewhere? In that case, can you just use a non-empty string for the initial label, e.g. "Lon:      , Lat:      "?  Or even "_"?

Reply:
nan just looks like a bug, even though it is accurate in this case (as there is no default).

Reply:
Don't think bokeh can do anything about that. The layout gets initialized with whatever is on the screen at the time, if there's no title to begin with it won't allocate the appropriate space for it and when one is added later it won't show up. We maybe be able to address it by triggering resize events.

Reply:
I wouldn't want a resize event to happen; that would be distracting.  But there's no need for a resize event, because Bokeh lines up by plot edge anyway, with or without a title, if one of the plots has a title:

![image](https://user-images.githubusercontent.com/1695496/31187469-a9a896a6-a8f7-11e7-8d33-cd536d2c77ce.png)

So I do think having the label later show up in the middle of the plot is a bug, not a natural consequence.  Anyway, your fix is fine for this PR, thanks!